client/plugins/conntrack/conntrack_linux.go

```diff
@@ -0,0 +1,68 @@
+package conntrack
+
+import (
+	"strconv"
+	"strings"
+
+	"k8s.io/klog/v2"
+	"k8s.io/utils/exec"
+	utilnet "k8s.io/utils/net"
+
+	v1 "sigs.k8s.io/kpng/api/localnetv1"
+)
+
+var execer = exec.New()
+
+func setupConntrack() {
+	// TODO
```

Comment:
do we need to setup anything before calling the conntrack module, or this is only a placeholder?

Reply:
there are things to do yes (upstream: cmd/kube-proxy/app/conntrack.go)